python/llm/test/benchmark/gpu/chatglm2.py

```diff
@@ -0,0 +1,85 @@
+#
```

Comment:
Can we use this script instead of using one script for one model?
https://github.com/intel-analytics/BigDL/pull/8871 @qiuxin2012

Reply:
@qiuxin2012 Can your script support Arc as well?